src/brevitas_examples/llm/main.py

```diff
@@ -462,6 +481,28 @@ def main(args):
             quant_ppl = compute_perplexity(
                 model, validation_loader, context_length=args.seqlen // 2, tokenizer=tokenizer)
         print(f"Quantized perplexity ({args.dataset}): {quant_ppl:.3f}")
+
+    if args.zero_shot_eval:
+        with torch.no_grad(), quant_inference_mode(model):
+            model(**calibration_loader[0])
+            if args.zero_shot_compile:
+                remove_hooks(model)
+                model.cuda()
+                model = torch.compile(model)
+
+            wrapped_model = HFLM(pretrained=model)  # need to wrap for LLM eval
+            results = evaluator.simple_evaluate(
+                model=wrapped_model,
+                model_args=None,
+                tasks=list(args.zero_shot_tasks),
+                device='cuda:0',
```

Comment:
Should this be `auto` for accelerate?

Reply:
https://github.com/EleutherAI/lm-evaluation-harness/blob/6ccd520f3fb2b5d74c6f14c05f9d189521424719/lm_eval/evaluator.py#L93

Looks like it only support torch devices. Even with accelerate, the execution device is always GPU 0, and that's where the data should go. Accelerate will take care of everything else under the hood.